src/services/ui/src/pages/dashboard/index.tsx

```diff
@@ -33,11 +35,20 @@ const loader = (queryClient: QueryClient) => {
     return isUser;
   };
 };
+
 export const dashboardLoader = loader;
 
 export const Dashboard = () => {
-  const { data: user } = useGetUser();
+  const navigate = useNavigate();
+  const userContext = useUserContext();
   const query = useOsQuery();
+  const { data } = useGetUser();
+
+  useEffect(() => {
+    if (data?.user?.["custom:cms-roles"] === "onemac-micro-statesubmitter") {
+      return navigate("/");
```

Comment:
I'm not sure if it's worth changing or not, but figured I'd throw a fun fact/tip in. React Router v6 has two ways to navigate! You can use the hook function, or the `<Navigate />` element. The way you've done it here is the former, and if you wanted to you could also do something like this:

```javascript
return data?.user?.["custom:cms-roles"] === "onemac-micro-statesubmitter" 
      ? <Navigate to={ROUTES.HOME} /> 
      : <OsProvider> ... </OsProvider>
```

Again, not sure there's a benefit of one over the other in this case, but it's a nifty tool!